mlflow/sklearn/utils.py

```diff
@@ -500,6 +499,7 @@ def _log_specialized_estimator_content(fitted_estimator, run_id, fit_args, fit_k
                     display.ax_.set_title(artifact.title)
                     filepath = tmp_dir.path("{}.png".format(artifact.name))
                     display.figure_.savefig(filepath)
+                    import matplotlib.pyplot as plt
```

Comment:
Sklearn's plotting functionality depends on matplotlib; accordingly, we do not anticipate that this import will fail